radixdlt-java/src/main/java/org/radix/utils/primitives/Longs.java

```diff
@@ -61,7 +61,7 @@ public static long fromByteArray(byte[] bytes) {
 	 * @param bytes The byte array to decode to a long
 	 * @param offset The offset within the array to start decoding
 	 * @return The decoded long value
-	 */
+	 * */

```

Comment:
Minor comment format fail.

Reply:
How did that even happen??? Anyways, fixed.